### DISCUSSION TITLE
How To Disable The Chatbot Self Conversation

### DISCUSSION THREAD

**Discussion by ZestyLemon01** [oobabooga/text-generation-webui#3418] [Q&A]
Currently, the chatbot that we have set up return a response to the question that was asked but it also provide new questions in the response and responding to that like a conversation. We would like to have the chatbot return a response to the question that was asked and nothing else, meaning not having the chatbot asking itself another question in the response. Attached below is an example of what is currently happening. I highlighted the issue in the image below and want to disable the conversation feature. 

![chatbox_image](https://github.com/oobabooga/text-generation-webui/assets/107714165/b07ac978-905d-4a56-8f21-7a5e0094771d)

**Comment by viperwasp** [comment]
I don't know if this will help or not but I do not like using these models in the form you use them in. I like using them in the chat or instruct modes. Were issues like this are less common. But I have only tried using the mode your showing once or twice. I did not like it. Maybe it is better though.

**Comment by oobabooga** [comment]
You can put "\nQuestion:" inside "Custom stopping strings" in the Parameters tab

**Reply by snyderpa** [reply]
I can't maintain a conversation with the model, I keep getting hijacked into a random conversation.  Is there a good how-to doc?

**Reply by snyderpa** [reply]
![Setting](https://github.com/oobabooga/text-generation-webui/assets/729818/46c81d8d-4cdc-41dc-9511-c70472ca0a56)
![Conversation](https://github.com/oobabooga/text-generation-webui/assets/729818/e7063c1e-4297-4885-bb1e-773ea4d1cd63)

**Comment by FartyPants** [comment]
There is no stopping in Default or Notebook. In chat mode it will stop generating when /s occurs.

**Reply by snyderpa** [reply]
That's good info but it's not enough to get me past this.  I was trying in chat mode with sub-settings of "instruct" and "chat-instruct", using "starchat-beta" model and prompts and having this problem.  Do I need to type "/s" somewhere?  Has anybody written a book or something?  This tool has many options unfriendly to a novice like me who wants a tool to privately query a pre-trained model.
